packages/lib/src/index.ts

```diff
@@ -24,6 +24,7 @@ export { default as FlipYAxisToggler } from './toolbar/controls/FlipYAxisToggler
 export { default as Selector } from './toolbar/controls/Selector/Selector';
 export { default as ExportMenu } from './toolbar/controls/ExportMenu';
 export { default as CellWidthInput } from './toolbar/controls/CellWidthInput';
+export { default as InteractionHelp } from './toolbar/controls/InteractionHelp';
```

Comment:
Maybe add an `InteractionHelp` story between the `Default` and `Responsive` toolbar stories? Especially if you refactor by adding an `interactions` prop to the `Toolbar`, which gets forwarded to `InteractionHelp`. Then, you can also pass an `interactions` object to the `Responsive` story to test that the button doesn't move to the overflow menu when space is limited.